set-client/src/main/java/eu/europeana/set/client/BaseUserSetApi.java

```diff
@@ -33,19 +27,7 @@ protected BaseUserSetApi(ClientConfiguration configuration) throws SetApiClientE
             throw new SetApiClientException(" Set Api Endpoint not provided !!!");
         }
 
-        if (this.configuration.getOauthServiceUri() == null || this.configuration.getOauthRequestParams() == null) {
-            throw new SetApiClientException("Oauth uri and param not provided !!!");
-        }
-
-        //get the real token in case the one is not provided
-        String regularUserTokenProvided=this.configuration.getOauthRegularUserToken();
-        String regularUserToken = regularUserTokenProvided!=null ? regularUserTokenProvided 
-            : getOauthToken(this.configuration.getOauthServiceUri(), this.configuration.getOauthRequestParams());
-
-        this.apiConnection = new UserSetApiConnection(
-                this.configuration.getServiceUri(),
-                this.configuration.getApiKey(),
-                regularUserToken);
+        this.apiConnection = new UserSetApiConnection(this.configuration.getServiceUri());
```

Comment:
we should still allow the set API to fetch OAuth tokens, only that it becomes optional when the token is provided with the configuration. consequntly the verification of oath endpoint and params is to be verified only if no oauth token is provided